errors

```diff
@@ -0,0 +1,203 @@
+/home/ubuntu/rfaas-refactor/rfaas/lib/resources.cpp: In member function ‘std::vector<int> rfaas::servers::select(int)’:
+/home/ubuntu/rfaas-refactor/rfaas/lib/resources.cpp:43:40: warning: unused parameter ‘cores’ [-Wunused-parameter]
+   43 |   std::vector<int> servers::select(int cores)
+      |                                    ~~~~^~~~~
+In file included from /home/ubuntu/rfaas-refactor/server/executor/server.hpp:15,
```

Comment:
Does this code even compile? What's the purpose of this file?

Reply:
I didn't mean to commit this file.